### PR TITLE
Improved Start Simulation example

### PR DESCRIPTION
**Pull request by fingeraugusto**

### CODE REVIEW

File: examples/basic_examples/start_simulation_async.py [hunk 1]
```diff
@@ -53,124 +44,145 @@
...
+    study.StartSimulation(non_blocking=True)
+
+    ###############################################################################
+    # Postprocess
+    # ~~~~~~~~~~~
+    # Obtain the in and out mass flows of the particles while the simulation is
+    # running.
+    particles = study.GetParticles()
+
+    import time
```
Comment (igortg) [inline examples/basic_examples/start_simulation_async.py:123]: Move to the top.

File: examples/basic_examples/start_simulation_async.py [hunk 2]
```diff
@@ -53,124 +44,145 @@
...
-# Obtain the in and out mass flows of the particles while the simulation is
-# running.
-particles = study.GetParticles()
-
-import time
+awp_roots = sorted(
+    [k for k in os.environ.keys() if k.startswith("AWP_ROOT")], reverse=True
+)
+last_rocky_version = awp_roots[0]
+if "25.1" in last_rocky_version:
```
Comment (igortg) [inline examples/basic_examples/start_simulation_async.py:51]: IMHO the previous version (early exit here) was better. In it we'd just remove the `if` when 25.1 becomes available.

**Review by igortg — APPROVED** [review]

File: examples/basic_examples/start_simulation_async.py (hunk 2 above)
Comment (fingeraugusto) [reply examples/basic_examples/start_simulation_async.py:51]: The problem is that that exit was breaking the documentation step. IMHO it was way better the other way.

**Review by alexandrelff — APPROVED** [review]

File: examples/basic_examples/start_simulation_async.py (hunk 2 above)
Comment (igortg) [reply examples/basic_examples/start_simulation_async.py:51]: Ah, Ok... Let's merge then.

**Pull request merged**